tests/xbmcgui.py

```diff
@@ -9,7 +9,7 @@
 from xbmcextra import kodi_to_ansi
 
 
-class Control:
+class Control:  # pylint: disable=too-few-public-methods
```

Comment:
This is one we prefer to add to .pylintrc, mostly because it is not a sane default IMO.

Reply:
Ok, this is wrong. Since this is in xbmcgui.py we don't want to do this in .pylintrc, sorry for the mixup.